src/test/scala/ai/eto/rikai/SparkTestSession.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 Rikai authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.eto.rikai
+
+import org.apache.spark.sql.SparkSession
+import org.scalatest.{BeforeAndAfterEach, Suite}
+
+trait SparkTestSession extends BeforeAndAfterEach {
+
+  this: Suite =>
+  lazy val spark: SparkSession = SparkSession.builder
```

Comment:
nit: the indent here looks off. Good thing it's not python

Reply:
should this line 2 more spaces indented?  This is the result auto-formatted by Intellij